cinema_service/settings.py

```diff
@@ -86,12 +91,26 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.getenv("POSTGRES_DB"),
+        "USER": os.getenv("POSTGRES_USER"),
+        "PASSWORD": os.getenv("POSTGRES_PASSWORD"),
+        "HOST": os.getenv("POSTGRES_HOST"),
+        "PORT": os.getenv("POSTGRES_PORT"),
     }
```

Comment:
You're using environment variables for your database settings, which is good for security. However, ensure your environment variables are properly set in your deployment environment. Also, it's a good practice to have a default value in case the environment variable is not set.